css/estilos.scss

```
@@ -97,19 +97,45 @@ body {
   color: #f0f0f0;
   transition: all .5s ease;
   z-index: 5;
-  button {
+  .close-lightbox {
     position: absolute;
     top: 3%;
     right: 3%;
+    cursor: pointer;
+    padding: 6px;
+    border-radius: 50%;
     background-color: #fff;
+    color: #444;
+    transition: .3s;
+  }
+  .close-lightbox:hover {
+    background-color: #444;
+    color: #fff;
   }
   .lightbox-data {
     width: 70%;
     height: 60%;
+    padding: 0;
+    border-radius: $radius;
     background: #444;
     display: flex;
     flex-direction: column;
     justify-content: flex-end;
+    background-size: cover;
+    background-position: center center;
+    background-repeat: no-repeat;
+    .lightbox-info {
+      background-color: rgba(0,0,0,.6);
+      .work-date {
+        margin: 0;
+      }
+      .work-description {
+        margin-bottom: 8px;
+      }
+      hr {
+        margin: 8px 0;

```

```suggestion
        margin: $radius 0;

```
En realidad no se hasta que punto es bueno usar las variables en ciertos casos, como este.  Quizá realmente no sea tan necesario usar la variable en casos específicos mas haya de que usen el mismo valor.